### PR TITLE
Motion Fixes

### PR DESCRIPTION
**Pull request by emmauss**
This the following issues reported with the motion client.

-  Slowdown caused by an overload of receiver tasks
- Alt Slot not being set.
- Crash when saving controller config with empty Port field

### CODE REVIEW

File: Ryujinx/Motion/Client.cs [hunk 1]
```diff
@@ -24,7 +24,8 @@ public class Client : IDisposable
         private readonly Dictionary<int, Dictionary<int, MotionInput>> _motionData;
         private readonly Dictionary<int, UdpClient> _clients;
 
-        private bool[] _clientErrorStatus = new bool[Enum.GetValues(typeof(PlayerIndex)).Length];
+        private bool[] _clientErrorStatus   = new bool[Enum.GetValues(typeof(PlayerIndex)).Length];
+        private long[] _clientRetryTimer = new long[Enum.GetValues(typeof(PlayerIndex)).Length];
```
Comment (LDj3SNuD) [inline Ryujinx/Motion/Client.cs:28]: Align.

File: Ryujinx/Motion/Client.cs [hunk 2]
```diff
@@ -163,16 +205,66 @@ private byte[] Receive(int clientId)
...
+                    {
+                        if (_client.Client.Connected)
+                        {
+                            try
+                            {
+                                while (_active)
                                 {
-                                    _clientErrorStatus[clientId] = false;
-                                }
+                                    byte[] data = Receive(clientId);
```
Comment (LDj3SNuD) [inline Ryujinx/Motion/Client.cs:257]: Maybe the level of indentation could be reduced.

**Review by LDj3SNuD — COMMENTED** [review]
I cannot approve since I cannot test.

**Review by Thog — APPROVED** [review]
lgtm

**Review by AcK77 — APPROVED** [review]

**Pull request merged**